blockchain/types/transaction_signing.go

```diff
@@ -56,8 +56,7 @@ type sigCachePubkey struct {
 func MakeSigner(config *params.ChainConfig, blockNumber *big.Int) Signer {
 	var signer Signer
 
-	// TODO-Klaytn-AccessList: Make another hardfork for EIP2930Signer instead of London
-	if config.IsLondon(blockNumber) {
+	if config.IsEthTxType(blockNumber) {
```

Comment:
Is this okay to substitute `IsLondone` with `IsEthTxType`? Don't we need another case for `IsLondon`?

Reply:
The london signer is only for new tx types that is introduced with EthTxType hardfork. So, it should be enabled with `IsEthTxType` check. 
I also think the naming is so confusing. Actually, londonCompatible means londonEVMCompatible.